scripts/synapse_sytest.sh

```diff
@@ -26,10 +26,10 @@ if [ -n "$MULTI_POSTGRES" ] || [ -n "$POSTGRES" ]; then
     # Start the database
     su -c 'eatmydata /usr/lib/postgresql/*/bin/pg_ctl -w -D $PGDATA start' postgres
 
-    su -c psql postgres <<< "show config_file"
-    su -c psql postgres <<< "show max_connections"
-    su -c psql postgres <<< "show full_page_writes"
-    su -c psql postgres <<< "show fsync"
+    #su -c psql postgres <<< "show config_file"
+    #su -c psql postgres <<< "show max_connections"
+    #su -c psql postgres <<< "show full_page_writes"
+    #su -c psql postgres <<< "show fsync"
```

Comment:
Should we remove these if they're not useful?

Reply:
done in 0165ef9b